template.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Cynkrablog Example
+author:
+  - David Schoch
+authors:
+  - David Schoch
+excerpt: |
+  A paragraph that is the excerpt
+date: 2025-04-09T00:00:00.000Z
+og_image: og_image.jpg
+toc_title: Table of Contents
```

Comment:
...also this comes from Maëlle, but i am not sure why. At the moment we do not have TOC. Here i cannot tag neither Maëlle nor Christoph, but maybe this can be removed.